ArgoTests/JSON/types.json

```diff
@@ -11,6 +11,7 @@
     "hello",
     "world"
   ],
+  "string_array_opt": null,
```

Comment:
Ah, no, I missed the fact that we were already trying to pull this key. Carry on.